packages/alfa-rules/src/sia-r62/rule.ts

```diff
@@ -344,25 +343,35 @@ function hasDistinguishableBackground(
 }
 
 /**
- * Check if an element has a different font weight than its container.
+ * Check if an element has a different font weight and family than its container.
  *
  * This is brittle and imperfect but removes a strong pain point until we find
  * a better solution.
  */
-function hasDistinguishableFontWeight(
+
+function hasDistinguishableFont(
   container: Element,
   device: Device,
   context?: Context
 ): Predicate<Element> {
-  const reference = Style.from(container, device, context).computed(
-    "font-weight"
-  ).value;
+  const style = Style.from(container, device, context);
 
-  return hasComputedStyle(
-    "font-weight",
-    not((weight) => weight.equals(reference)),
-    device,
-    context
+  const referenceWeight = style.computed("font-weight").value;
+  const referenceFamily = style.computed("font-family").value;
+
+  return or(
+    hasComputedStyle(
+      "font-weight",
+      not((weight) => weight.equals(referenceWeight)),
+      device,
+      context
+    ),
+    hasComputedStyle(
+      "font-family",
+      not((family) => family.equals(referenceFamily)),
```

Comment:
We should only look at the first in the list, not at everything. So that `font-family: some-font, foo` and `font-family: some-font, bar` are considered the same (under the assumption that `some-font` loads, of course).